staging/src/k8s.io/apiserver/pkg/storage/cacher/watch_cache.go

```diff
@@ -44,6 +44,13 @@ const (
 	// resourceVersionTooHighRetrySeconds is the seconds before a operation should be retried by the client
 	// after receiving a 'too high resource version' error.
 	resourceVersionTooHighRetrySeconds = 1
+
+	// eventRecordDuration is time duration of events we want to keep.
+	eventFreshDuration = 5 * time.Minute
+
+	// maxEventCapacity is a heuristic value of cache capacity upper bound. As leases is most frequent resource happens per 10s.
```

Comment:
We don't know if leases are the most frequent ones - it depends on what users are actually doing.

Reply:
replace it by defaultUpperBoundCapacity, but defaultUpperBoundCapacity and defaultLowerBoundCapacity don't have more elaborate comments now.